.github/workflows/ci.yml

```diff
@@ -29,6 +29,22 @@ env:
   CI_DEVICE_ADVISOR: ${{ secrets.AWS_CI_DEVICE_ADVISOR_ROLE }}
 
 jobs:
+  all-python-versions:
```

Comment:
There's a failing check: `AWS CodeBuild us-east-1 (CI_Python_V2_LinuxSmokeTests)`

Also, I'm noticing that a lot of the checks aren't `required` right now ... could you please make every check required that ought to be required?

Reply:
Flipped them to required. The currently failing check is expected to fail until we merge https://github.com/aws/aws-iot-device-sdk-python-v2/pull/366 that shifts the codebuild jobs to the IoT account. I believe the two codebuild jobs are checking the same thing, just using different secrets.